daemon/resize_test.go

```diff
@@ -13,7 +13,9 @@ import (
 
 // This test simply verify that when a wrong ID used, a specific error should be returned for exec resize.
 func TestExecResizeNoSuchExec(t *testing.T) {
-	n := "TestExecResize"
+	n := t.Name()
+	const width = 24
+	const height = 8
```

Comment:
```suggestion
	const (
		width = 24
		height = 8
	)
```

Reply:
Ah, yes, that's slightly nicer; usually `gofumpt` is picky on these, but I think it has a threshold (more than X consts).

I'll update it though!